etna/articles/migrations/0009_add_author_block.py

```diff
@@ -25,7 +26,7 @@ class Migration(migrations.Migration):
                                 (
                                     "author",
                                     wagtail.snippets.blocks.SnippetChooserBlock(
-                                        "authors.Author"
+                                        "people.Author"
```

Comment:
Editing previous migrations is an anti-pattern and quite risky.

Could we just have new migrations on top of the existing ones?

Reply:
I was getting an error after I had updated the app name from `authors` to `people` because the app `authors` didn't exist anymore - it was blocking the site running, as well as the migrations

I've just changed it back now and I'm not getting any errors - it might be okay for us to keep these as `authors` 🤞

Reply:
https://github.com/nationalarchives/ds-wagtail/actions/runs/10264284241/job/28397992995?pr=1698

It errors if we don't change the app name in the migrations ☹️ 

Here is a PR from the last time we did it: https://github.com/nationalarchives/ds-wagtail/pull/601